### DISCUSSION TITLE
Is it possible leave modules structure?

### DISCUSSION THREAD

**Discussion by aspirisen** [webpack/webpack#15730] [Q&A] [answered]
Hello, i am working on a library and currently I build it by gulp, because I can process each module and have the same modules structure. For example apply css modules rules, transform TS to JS, convert SVG to React components etc. I wonder if it is possible to have the same behaviour using webpack, not to create single file but leave the files structure? 
Thanks.

**Comment by vankop** [comment, marked as answer]
>  I wonder if it is possible to have the same behaviour using webpack, not to create single file but leave the files structure?

no, webpack creates a bundle ( it could have several chunks of code )
